modules/n1ql/pages/n1ql-language-reference/curl.adoc

```diff
@@ -519,7 +519,7 @@ FROM CURL("http://localhost:8093/query/service",
     "user": "Administrator:password"}).results[*].t route
 JOIN `travel-sample`.inventory.airline
 ON KEYS route.airlineid
-LIMIT 4;
+LIMIT 4
```

Comment:
This semicolon doesn't need to go, because it's part of the outer query, not part of the query within the CURL() function

It doesn't actually matter if you do remove it though; I'd leave it in, for consistency with the other examples in this page

Reply:
Did this as a suggestion commit.